pkg/storage/cacher_test.go

```diff
@@ -221,6 +221,7 @@ func TestWatch(t *testing.T) {
 	_ = updatePod(t, etcdStorage, podBar, nil)
 	fooUpdated := updatePod(t, etcdStorage, podFooPrime, fooCreated)
 
+	t.Log("verifying")
```

Comment:
I was really trying to make the minimal change to the test to make it easy to review. I can make a bigger change if you prefer, but marking where we were in the parent test was the least disruptive way to make failures in the verify function understandable